rulesets/guardian-securedrop-ruleset.xml

```diff
@@ -0,0 +1,5 @@
+<ruleset name="The Guardian">
+	<target host="www.theguardian.securedrop.tor.onion" />
```

Comment:
Ah.. I'm realizing we have a shortcoming in the `sddir.py` script and will need to edit it. Basically I wrote the `sddir.py` script back when we were merely stripping the original TLD and adding the pseudo-TLD, but now we're letting organizations choose a human-readable name (providing it will not collide with another news organization or cause confusion). 

As such, I think we need to store in version control _both_ (i) the landing page domain (in this case `www.theguardian.com` as you correctly have in `onboarded.txt`) such that we can join with the relevant SD directory API entry, and (ii) the desired rule, since we'll want to use that for the `target host` and `rule from` fields when writing this rule in `rulesets/`. I'm thinking maybe something like making `onboarded.txt`  CSV with fields `primary_domain, sd_rewrite_url` (one row per organization) and then modifying the `sddir.py` script to use it. What do you think @kushaldas? Feel free to deviate from this if you have another idea.